# Config file for demo purposes

buoy_publisher:
  ros__parameters:
    buoy_locations_x: [5.0, -5.0, 5.0, -5.0]
    buoy_locations_y: [5.0, 5.0, -5.0, -5.0]
    buoy_ranges: [2.0, 5.0, 5.0, 2.0]
    pose_uncertainty_x: [0.0, 0.0, 0.0, 0.0]
    pose_uncertainty_y: [0.0, 0.0, 0.0, 0.0]
    pose_uncertainty_z: [0.0, 0.0, 0.0, 0.0]
    buoy_color: [1.0, 0.39, 0.0, 1.0]
    radius_color: [0.0, 1.0, 0.0, 0.25]
    buoy_marker_scale: 2.0
    header_frame_id: "map"